plugins/modules/site_workflow_manager.py

```diff
@@ -760,14 +992,266 @@ def get_want(self, config):
             parameters such as 'site_params' and 'site_name.' The gathered
             information is stored in the 'want' attribute for later reference.
         """
+        try:
+
+            want_list = []
+            self.log(self.pprint(self.config))
+            if 'site' in self.config[0]:
+                if self.get_ccc_version_as_integer() >= self.version_2_3_7_6:
+                    self.keymap = {}
+                    self.keymap = self.map_config_key_to_api_param(self.keymap, config)
+                    self.keymap.update({
+                        "floor_number": "floorNumber",
+                        "rf_model": "rfModel",
+                        "parent_name_hierarchy": "parentNameHierarchy",
+                        "units_of_measure": "unitsOfMeasure"
+                    })
+                    self.log(self.keymap, "INFO")
```

Comment:
Same add some msg before printing the variable value. Please take care at all the places in the module.

Reply:
Added